snappy_pipeline/workflows/helper_gcnv_model/__init__.py

```diff
@@ -230,6 +231,28 @@ def get_cnv_model_result_files(self, _unused=None):
             library_kit=chosen_kits,
         )
 
+    def get_resource_usage(self, action):
+        """Get Resource Usage
+
+        :param action: Action (i.e., step) in the workflow, example: 'run'.
+        :type action: str
+
+        :return: Returns ResourceUsage for step.
+        """
+        result = super().get_resource_usage(action)
+
+        def get_memory(wildcards, input=None, threads=None, attempt=None):
```

Comment:
Can we unit test this for `filter_intervals` at least?

Reply:
OK, added a test. Please review